nav2_waypoint_follower/include/nav2_waypoint_follower/plugins/wait_at_waypoint.hpp

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2020 Fetullah Atas
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_WAYPOINT_FOLLOWER__PLUGINS__WAIT_AT_WAYPOINT_HPP_
+#define NAV2_WAYPOINT_FOLLOWER__PLUGINS__WAIT_AT_WAYPOINT_HPP_
+#pragma once
+
+#include <string>
+
+#include "rclcpp/rclcpp.hpp"
+#include "rclcpp_lifecycle/lifecycle_node.hpp"
+#include "nav2_core/waypoint_task_executor.hpp"
+
+namespace nav2_waypoint_follower
+{
+
+/**
+ * @brief Simple plugin based on WaypointTaskExecutor, lets robot to sleep for a
+ *        specified amount of time at waypoint arrival. You can reference this class to define
+ *        your own task and rewrite the body for it.
+ *
+ */
+class WaitAtWaypoint : public nav2_core::WaypointTaskExecutor
+{
+private:
```

Comment:
Move everything under `private` to the bottom. We usually do `public` functions, then `protected` variables. We want `protected` in case someone wants to subclass from this and wants to access those variables/.